### PR TITLE
Developers: add xLINK contracts documentation (w1)

### PR DESCRIPTION
**Pull request by ignaciopenia**
## Description 

Added xLINK contract technical documents. Impacts on https://docs.xlink.network/. This branch is live at a temporary URL [here ](https://coinfabrik.gitbook.io/xlink-docs/xlink-contracts-w1).

### Changes

1. Created the Developers section (see Gitbook page groups).
2. Added technical documentation for the contracts:
    
    - `btc-peg-in-endpoint-v2-03.clar`
    - `meta-peg-in-endpoint-v2-02.clar`
    
3. Updated the `SUMMARY.md` accordingly. 

### Following work

- Continue with the contracts technical documentation as follows: `btc-peg-out-endpoint-v2-01.clar`, `meta-peg-out-endpoint-v2-03.clar`, `cross-peg-in-endpoint-v2-03.clar`, `cross-peg-out-endpoint-v2-01.clar`.
- Index page and comprehensive overview for the technical documentation of contracts.
- Design refinements to accommodate the new sections on GitBook.

### CODE REVIEW

File: developers/contracts/btc-peg-in-endpoint-v2-03.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+| Variable | `bool` |
+
+A flag that indicates whether the peg-in process is active. If set to true, all peg-in operations are paused, preventing any new transactions. The contract is deployed in a paused state by default.
+
+### `peg-in-fee`
+| Data     | Type   |
+| -------- | ------ |
+| Variable | `uint` |
+
+The percentage fee charged for peg-in transactions. By default, this value is u0.
```
Comment (tenuki) [inline developers/contracts/btc-peg-in-endpoint-v2-03.md]: here u0 must be `u0`

**Review by tenuki — CHANGES_REQUESTED** [review]
one minor change.!

**Review by tenuki — CHANGES_REQUESTED** [review]
A minor change is required/requested.

File: developers/contracts/btc-peg-in-endpoint-v2-03.md (hunk 1 above)
Comment (ignaciopenia) [reply developers/contracts/btc-peg-in-endpoint-v2-03.md]: Done

**Review by tenuki — APPROVED** [review]

**Pull request merged**